### PR TITLE
Change default autoconvert behaviour

### PR DESCRIPTION
**Pull request by spookylukey**
See issue #199

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -1,6 +1,9 @@
 Changes in 0.8.1
 ----------------
 - Added Django 1.10 support `198 <https://github.com/django-money/django-money/issues/198>`_ (`Dmitry Dygalo <https://github.com/Stranger6667>`_)
+- Changed auto conversion of currencies using djmoney_rates (added in 0.7.3) to
```
Comment (benjaoming) [inline CHANGES.rst]: With this change in mind, would you mind bumping the version for all the above unreleased changes to 0.9 (changing the subheading)

The version itself is bumped in setup.py

**Comment by benjaoming** [comment]
Looks ripe! Great stuff @spookylukey !

File: CHANGES.rst (hunk 1 above)
Comment (benjaoming) [inline CHANGES.rst]: *doh* I already did the version bump in #201 

Sorry if you're getting conflicts now........

File: djmoney/models/fields.py [hunk 2]
```diff
@@ -34,12 +34,17 @@
...
     except ImportError:
         # NOTE. djmoney_rates doesn't support Django 1.9+
-        AUTO_CONVERT_MONEY = False
+        RATES_INSTALLED = False
 else:
-    AUTO_CONVERT_MONEY = False
+    RATES_INSTALLED = False
+
+
+if getattr(settings, 'DJMONEY_AUTO_CONVERT_MONEY', False):
```
Comment (Stranger6667) [inline djmoney/models/fields.py]: This line is duplicated (at line 79), so another constant with ``getattr(settings, 'DJMONEY_AUTO_CONVERT_MONEY', False)`` will be useful

File: djmoney/models/fields.py [hunk 3]
```diff
@@ -71,7 +76,7 @@ def _convert_to_local_currency(self, other):
         """
         Converts other Money instances to the local currency
         """
-        if AUTO_CONVERT_MONEY:
+        if getattr(settings, 'DJMONEY_AUTO_CONVERT_MONEY', False):
```
Comment (benjaoming) [inline djmoney/models/fields.py]: Something like `djmoney.settings.AUTO_CONVER_MONEY_ENABLED` would be cool?

File: djmoney/models/fields.py (hunk 2 above)
Comment (spookylukey) [inline djmoney/models/fields.py]: I did it that way at first, but then you end up with a module level constant, and can't change it (easily) in the tests, at least, not with the normal 'settings' fixture provided by pytest-django - https://pytest-django.readthedocs.io/en/latest/helpers.html#settings - you would have to do some other kind of monkey patching of that constant. I thought this duplication was the lesser of two evils. What do you think?

File: CHANGES.rst (hunk 1 above)
Comment (spookylukey) [inline CHANGES.rst]: I didn't do anything on this yet - are you saying we don't need a version bump? The latest released version is 0.8 right?

Also are you OK with the code duplication above for the reasons given?

Comment (benjaoming) [inline CHANGES.rst]: It's because I've merged another PR meanwhile that bumped the version, so you'll need to fix a conflict here...

Comment (spookylukey) [inline CHANGES.rst]: OK, fixed this now by merging from master.

**Comment by benjaoming** [comment]
Rebased in #207

**Pull request closed without merging**